setup.py

```diff
@@ -302,7 +302,7 @@ def write_version(filename: str = os.path.join(*[my_dir, "airflow", "git_version
     'google-cloud-build>=3.0.0,<4.0.0',
     'google-cloud-container>=0.1.1,<2.0.0',
     'google-cloud-datacatalog>=3.0.0,<4.0.0',
-    'google-cloud-dataproc>=2.2.0,<2.6.0',
+    'google-cloud-dataproc>=3.0.0,<4.0.0',
```

Comment:
It looks like a breaking chhange. Can you add a note about it in the changelog for the provider?

Reply:
@mik-laj 👍

Reply:
```suggestion
    'google-cloud-dataproc>=2.2.0,<4.0.0',
```
We don't need to change the lower bound. New API is also available in 2.* series. In this way, we will avoid breaking changes.

Reply:
👍